Editor/Menu/VirtualMenu.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System;
+﻿#if MA_VRCSDK3_AVATARS
```

Comment:
I wonder if it would be better to move the menu processing (or rather, the VRChat-specific stuff) to a separate asmdef and make it conditionally built...

Reply:
I think breaking up asmdef would deserve a separate PR before this one. I am not certain enough how to move assets into folders, though. (at least I can try... everything VRCSDK under `Runtime/VRCSDK` and `Editor/VRCSDK` I suppose?)

Reply:
Couldn't try because extracting VRChat asmdef required extracting PluginDefinitions asmdef, too much to include in this PR

Reply:
Fair enough. We can work on that in a followup (in any case, I don't think we can move components...)